server/Controllers/FridgeController.cs

```diff
@@ -1,12 +1,35 @@
 ﻿using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using server.Models;
+using Microsoft.EntityFrameworkCore;
+using server.Data;
+using server;
 
 namespace server.Controllers
 {
     [ApiController]
     [Route("api/[controller]")]
     public class FridgeController : ControllerBase
     {
-        public FridgeController() { }
+
+        private readonly ApplicationDbContext _context;
+        public FridgeController(ApplicationDbContext context) 
+        {
+            _context = context;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Test()
+        {
+            var ingredients = await _context.Ingredient.ToListAsync();
+            if (ingredients != null)
+            {
+                return Ok(ingredients);
```

Comment:
Return this right away. `ToListAsync` returns an empty list if no ingredients are found and that should be the desired behavior.